cmake/thirdparty/dawnAddprotobuf.cmake

```diff
@@ -0,0 +1,40 @@
+##===------------------------------------------------------------------------------*- CMake -*-===##

```

Comment:
Inconsistent file name, should be **D**awnAddProtobuf

Or maybe just don't prefix them at all as I think later on you want to all these `AddX` files in repository and reuse them for all projects (in this case you need to clean them up to get rid of all gtclang and dawn references)

Reply:
I did rename it. 
But are u sure we should move it to msbuild? it is just a convenient indirection that specifies the version of protobuf, but other than that the main function is find_package. Different packages might require different versions of protobuf